dspace/config/modules/orcid.cfg

```diff
@@ -0,0 +1,24 @@
+#------------------------------------------------------------------#
```

Comment:
Since this is (mostly) and authentication plugin, we should name this `authentication-orcid.cfg`, similar to `authentication-shibboleth.cfg` (and other authentication plugins).